configs/dynamic_rcnn/dynamic_rcnn_r50_fpn_1x.py

```diff
@@ -0,0 +1,117 @@
+_base_ = [
+    '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+# model settings
```

Comment:
We may directly inherit `../faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py`.

Reply:
Fixed.